### PR TITLE
Added LatLng to the PollingLocation object.

### LINKED ISSUE #165: Include lat/long for polling locations
The second comment from the states:

> I would suggest the ability to submit “XY” latitude/longitude coordinates for polling locations for those who collect that information (NB: we use decimal degrees to seven significant figures). We find that using polling location coordinates is more reliable and more accurate than traditional address interpolation.

This did exist in the never-launched 4.0 specification. It may be worth adding this to `ContactInformation` as well rather than just polling location in case it could be useful for other elements (e.g. `ElectionAdministration`). To that point, does it make sense to extract the address "guts" of `ContactInformation` and reuse in `PollingLocation` (NB: I have a feeling we've probably had a lengthy conversation about this...but I don't remember)?

### PR DESCRIPTION
**Pull request by jdmgoogle**
This should fix #165.

### CODE REVIEW

**Comment by jungshadow** [comment]
Perfect.

**Comment by jungshadow** [comment]
Pinging @pstenbjorn, @cjerdonek, @jktomer, et al for sign-off.

**Comment by pstenbjorn** [comment]
Looks good to me.  Thanks

File: docs/xml/elements/polling_location.rst [hunk 1]
```diff
@@ -50,12 +50,47 @@ The Polling Location object represents a site where voters cast or drop off ball
 |                       |                       |              |            |polling location is |the implementation is  |
 |                       |                       |              |            |an early vote site. |required to ignore it. |
 +-----------------------+-----------------------+--------------+------------+--------------------+-----------------------+
+| LatLng                | :ref:`LatLng          | Optional     | Single     |**LatLng** specifies|If **LatLng** is       |
+|                       | <pl-latlng>`          |              |            |the latitude and    |invalid or not present,|
```
Comment (jktomer) [inline docs/xml/elements/polling_location.rst]: > **LatLng** specifies the latitude and of this polling location.

you accidentally a longitude

**Comment by jktomer** [comment]
LGTM

**Comment by jdmgoogle** [comment]
Thanks. Merging.

**Pull request merged**